starport/templates/typed/indexed/stargate.go

```diff
@@ -395,26 +395,23 @@ import "%s/%s.proto";`
 		templateMessages := `%[1]v
 message MsgCreate%[2]v {
   string creator = 1;
-%[3]v
-%[4]v}
-message MsgCreate%[2]vResponse { }
+%[3]v}
+message MsgCreate%[2]vResponse {}
 
 message MsgUpdate%[2]v {
   string creator = 1;
-%[3]v
-%[4]v}
-message MsgUpdate%[2]vResponse { }
+%[3]v}
+message MsgUpdate%[2]vResponse {}
 
 message MsgDelete%[2]v {
   string creator = 1;
-%[3]v
-}
-message MsgDelete%[2]vResponse { }
+%[4]v}
+message MsgDelete%[2]vResponse {}
 `
 		replacementMessages := fmt.Sprintf(templateMessages, typed.PlaceholderProtoTxMessage,
 			opts.TypeName.UpperCamel,
-			indexes,
 			fields,
+			deleteFields,
```

Comment:
I was trying another implementation before, and I changed it, but I thought to keep this part because it's clearer for me. I will revert for also not put in the same PR